### PR TITLE
docs: change the ID of the new documentation page

### PR DESCRIPTION
**Pull request by italicize**
Fix two links on **[Create Pages](https://docusaurus.io/docs/en/next/tutorial-create-pages)**.

### CODE REVIEW

File: docs/tutorial-create-pages.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = HelloWorld;
 
 > Use any text editor to make the file, such as [Microsoft Visual Studio Code](https://code.visualstudio.com/download) or [Komodo Edit](https://www.activestate.com/komodo-edit).
 
-2. Go to http://localhost:3000/hello-world and you should be able to see the new page.
+2. Go to http://localhost:3000/docusaurus-tutorial/hello-world and you should be able to see the new page.
```
Comment (yangshun) [inline docs/tutorial-create-pages.md]: This is only true after you have proceeded to add the `baseUrl` isn't it? When it's a fresh clone you don't see this.

Comment (italicize) [reply docs/tutorial-create-pages.md]: @yangshun You're right. I removed `docusaurus-tutorial/` from the links.

**Comment by yangshun** [comment]
@italicize Why doc4 to doc9?

**Comment by italicize** [comment]
@yangshun The ID for the new page cannot be `doc4`, because that ID is used by the page exampledoc4.md (which is downloaded by docusaurus-init). The ID `doc9` is only a suggestion.

**Pull request merged**